lomond/session.py

```diff
@@ -115,13 +115,38 @@ def _connect(self):
         sock.setsockopt(socket.IPPROTO_TCP, socket.TCP_NODELAY, 1)
         sock.settimeout(30)  # TODO: make a parameter for this?
         if self.websocket.is_secure:
-            sock = ssl.wrap_socket(sock)
+            sock = self._wrap_socket(sock)
         sock.connect(self._address)
         # The timeout makes the socket non-blocking
         # We want to the socket to block after the connection
         sock.settimeout(None)
         return sock
 
+    def _wrap_socket(self, sock):
```

Comment:
Looks like a fairly straightforward change to add SNI support. Documentation was thin on the ground.